src/contexts/addresses.tsx

```diff
@@ -432,10 +439,6 @@ export const AddressesContextProvider: FC<{ overrideContextValue?: PartialDeep<A
   useEffect(() => {
     // In case the "to" address of a pending transaction is an address of this wallet, we need to query the API for this
     // one as well
-    const addressesWithPendingSentTxs = addressesOfCurrentNetwork.filter(
-      (address) => address.transactions.pending.filter((pendingTx) => pendingTx.network === currentNetwork).length > 0
-    )
-
```

Comment:
There's no point having this here, it already exists outside of this `useEffect`. It will evaluate to the same value.

Reply:
But they should give the same result.

Reply:
If you've found this to be the case then ok - I did this under the assumption it's possible (and it is possible) that the statement inside the useEffect can evaluate to something else because of the nature of when useEffect fires.